### DISCUSSION TITLE
Contributing pattern use cases and sharing final design examples

### DISCUSSION THREAD

**Discussion by ctecks** [twilio-labs/paste#1593] [Ideas] [answered]
Idea that came out of Q2's Content Advocacy Program feedback.

**Problem/ask** 
PDs want more "real world" examples of content in action in patterns and components. They also want to see how patterns and components are being used to solve certain user problems to inform their decisions - in particular if the work has been validated/user tested/UX Reviewed.

**Context / Considerations**
The Paste-hosted /content guidance isn't regularly updated, nor is UDL patterns work sufficiently supported by content due to lack of capacity. 

**Hypothesis** 
Supplementing pattern/component documentation with links to/screenshots of the element in use, a brief description of the use case, context, and user feedback would be valuable to PDs coming to the element for the first time. 

In terms of time and effort, it _should_ be a light lift as the documentation and designs already exist, the PD would just be making the connection. 

The impact could be significant and at scale:

- making it more efficient for PDs to understand and decide which patterns to use and how
- supporting consistent usage of Paste
- modeling consistent, approved content in context
- increasing visibility for PDs of other PDs' work.

**Proposed solutions**
TBD - open for discussion. If this feels valuable, I'd like to discus ways of making it work so that it's effort and time light. Ideas:

- the contribution of approved designs to documentation is templated to make it swifter (I can support on this if useful)
- the contribution of approved designs becomes part of the "end of project" process for PDs 
- a regular call (quarterly?) goes out to PDs asking for Paste usage contributions to documentation
- other...

**Comment by serifluous** [comment]
@ctecks This is an amazing idea!

Creating a template would be great. Since there's already dsys and content review in UX reviews, that could also be a good channel for getting some designs from to start. We've definitely been interested in creating some sort of lighter weight way for people to contribute patterns/designs without the bigger lift of defining all the guidelines.

@zahnster has also been chatting about creating an "Examples" section for the site, which would be great even just to have one single resource for approved designs.

...Maybe a Think Week project? 👀 Would love to work on this with you.

**Reply by ctecks** [reply]
Sounds great! Thanks for the speedy response and enthusiasm. Would love to work on this together. Eventually, a repository of designs searchable by use case/pattern would be fairly stellar, but I appreciate that's a lift. 

I'm on PTO next week, but can draft up a template during think week and post here for async thoughts/comments. I have no doubt it'll evolve as folks use it, and rightly so, but my main aim is to get a tool out there to start the process of sharing work.

Very open to further suggestions for how the contribution process might be dovetailed into something that exists already.

**Comment by TheSisb** [comment, marked as answer]
Hey @ctecks,

We're piloting a new idea on DSYS for something very similar to this idea. They're kind of like Moodboards, but on Figma.  Here's the template: https://www.figma.com/file/6MlUZI1JHp8aIWE5JCGlm0/Paste-Moodboard?node-id=10%3A16879

![image](https://user-images.githubusercontent.com/1592327/127198925-8b21d0b0-bd25-4fd1-b4c7-34bc23d0fae7.png)
You can see other moodboards in the sidebar, and if one you want is missing you can right click to duplicate the template. We're thinking of using this as a place to gather data before tackling new patterns and components. Please let us know what you think!

**Reply by ctecks** [reply]
Love this idea for developing patterns - great way to leverage Figma - have you considered a "reviewed/tested" category for work that's using existing patterns and is live or has been through user testing/UX Review successfully? It's the latter I'm trying to get sorted - a repository for work that's had the stamp of approval and folks can look to with a degree of confidence when making decisions.
